cypress/integration/Modal.spec.ts

```diff
@@ -409,22 +408,22 @@ describe('Modal', () => {
           });
         });
 
-        it('should transfer focus to the header element', () => {
-          cy.findByRole('dialog', {name: 'Delete Item'})
-            .pipe(h.modal.getTitle)
+        it('should transfer focus to the cancel button', () => {
+          cy.findByRole('button', {name: 'Cancel'})
             .should('have.focus');
         });
 
         it('should trap focus inside the modal element', () => {
           cy.findByRole('dialog', {name: 'Delete Item'})
-            .pipe(h.modal.getTitle)
+            .findByRole('button', {name: 'Cancel'})
```

Comment:
This just moves up to the comment I mentioned above.

Reply:
The whole section should look like this: 

```tsx
cy.findByRole('button', {name: 'Cancel'}).should('have.focus');
 cy.tab()
 .should('contain', 'Delete')
 .tab()
 .should('contain', 'Delete Item')
 .tab()
 .should('contain', 'Cancel');
cy.focused().should('have.text', 'Cancel');
```

Reply:
I think I finally got it!